js/views/ListingCard.js

```diff
@@ -372,7 +372,7 @@ export default class extends baseVw {
   remove() {
     if (this.fullListingFetch) this.fullListingFetch.abort();
     if (this.destroyRequest) this.destroyRequest.abort();
-    $(document).off(null, this.boundDocClick);
```

Comment:
Didn't realize jQuery's off requires this param. It's optional on Backbone's version, in which case the given handler would be removed for all event types.